bin/webui.py

```diff
@@ -114,6 +114,7 @@ def mod_python_handler(r):
         415: apache.HTTP_UNSUPPORTED_MEDIA_TYPE,
         416: apache.HTTP_RANGE_NOT_SATISFIABLE,
         417: apache.HTTP_EXPECTATION_FAILED,
+        418: apache.HTTP_IM_A_TEAPOT,
```

Comment:
I think we need to take this out; AFAICT your patch to mod_python has been accepted, but they haven't released yet, so this would cause errors for anyone using mod_python until it's available. Also, REDbot (for better or worse) doesn't actually generate 418, so this won't ever be used.